acds/archetypes/ron.py

```diff
@@ -43,10 +43,11 @@ def __init__(
         dt: float,
         gamma: Union[float, Tuple[float, float]],
         epsilon: Union[float, Tuple[float, float]],
+        diffusive_gamma: float,
```

Comment:
consider adding a default value for this, otherwise it breaks previous usages which did not require the diffusive term

Reply:
I have already put a default value for diffusive gamma in the experiments, but it's true that without the default value here, it breaks previous usages not implemented in my thesis. So, I correct this error, which also resolves the next issue, as the diffusive matrix would be filled with 0 if not used.